src/main/java/appeng/capabilities/Capabilities.java

```diff
@@ -0,0 +1,48 @@
+package appeng.capabilities;
+
+
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.CapabilityInject;
+import net.minecraftforge.common.capabilities.CapabilityManager;
+
+import appeng.api.storage.IStorageMonitorableAccessor;
+
+
+/**
+ * Utility class that holds various capabilities, both by AE2 and other Mods.
+ */
+public class Capabilities
+{
+
+	@CapabilityInject( IStorageMonitorableAccessor.class )
+	public static Capability<IStorageMonitorableAccessor> STORAGE_MONITORABLE_ACCESSOR;
+
+	/**
+	 * Register AE2 provided capabilities.
+	 */
+	public static void register()
+	{
+		CapabilityManager.INSTANCE.register( IStorageMonitorableAccessor.class, createNullStorage(), NullMENetworkAccessor::new );
+	}
+
+	// Create a storage implementation that does not do anything
+	private static <T> Capability.IStorage<T> createNullStorage()
+	{
+		return new Capability.IStorage<T>()
+		{
+			@Override
+			public NBTBase writeNBT( Capability<T> capability, T instance, EnumFacing side )
+			{
+				return null;
+			}
+
+			@Override
+			public void readNBT( Capability<T> capability, T instance, EnumFacing side, NBTBase nbt )
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)